book/src/artifacts/archives.md

```diff
@@ -0,0 +1,93 @@
+# Archives
+
+Archives are the primary output of cargo-dist: a zip (or tarball) containing prebuilt executables/binaries for an app, along with additional static files like READMEs, LICENSES, and CHANGELOGs. The docs previously referred to these as "executable-zips", so if you ever see that term floating around, this is what's being talked about.
```

Comment:
👍 "executable-zip" is perfectly fine as an implementation name but I don't think it was the best thing to advertise to users. I think this is a nicer name to advertise.